docs/tutorials/vehicle_model/README.md

```diff
@@ -0,0 +1,242 @@
+# Tutorial: Create a vehicle model with DTDL
+
+- [Introduction](#introduction)
+- [1. Create a reference vehicle model with DTDL](#1-create-a-reference-vehicle-model-with-dtdl)
+  - [1.1 DTDL Interfaces](#11-dtdl-interfaces)
+  - [1.2 Create HVAC and HMI interfaces](#12-create-hvac-and-hmi-interfaces)
+  - [1.3 DTDL DTMI ID](#13-dtdl-dtmi-id)
+  - [1.4 DTDL properties](#14-dtdl-properties)
+  - [1.5 DTDL commands](#15-dtdl-commands)
+- [2. DTDL validation](#2-dtdl-validation)
+- [3. Translating DTDL to code](#3-translating-dtdl-to-code)
+
+## Introduction
+
+In this tutorial, you will learn how to create a vehicle model using [Digital Twins Definition Language (DTDL) version 3.0](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html), and build a corresponding Rust vehicle model. The focus will be on translating DTDL to a programming language, specifically Rust. While Rust is used here to be consistent with Ibeji's samples being in Rust, you can use any programming language.
+
+This tutorial provides a basic understanding of DTDL in the context of our Ibeji samples, but it is not a comprehensive guide to DTDL. Specifically, it covers [interfaces](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#interface), [commands](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#command), [properties](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html#property)
+
+Please refer to the [DTDL v3](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3.html) documentation for more information on DTDL.
+
+>Note: Currently in Ibeji, DTDL is used as a reference representation of the vehicle hardware. It serves as a guide to manually construct the vehicle model in code. In the future, the vehicle model code should be generated from a DTDL file.
+
+## 1. Create a reference vehicle model with DTDL
+
+In this section, you will be learning how to create a reference vehicle model with DTDL. DTDL is used to define a digital twin model of a vehicle, and its capabilities.

```

Comment:
Try to use active language where possible. There's also an extraneous comma here:

```suggestion
In this section, you will learn how to create a reference vehicle model with DTDL. DTDL is used to define a digital twin model of a vehicle and its capabilities.

```